src/entries/popup/pages/home/TokenDetails/TokenContextMenu.tsx

```diff
@@ -88,7 +88,7 @@ export function TokenContextMenu({ children, token }: TokenContextMenuProps) {
             {`${i18n.t('token_details.swap')} ${token.symbol}`}
           </ContextMenuItem>
         )}
-        {isBridgeable && (
+        {!isWatchingWallet && isBridgeable && (
```

Comment:
this fixes another bug, where you could right click and select to bridge a token in a watched wallet